src/types.ts

```diff
@@ -1 +1,2 @@
 export class AuthenticationError extends Error {}
+export class DbOperationError extends Error {}
```

Comment:
Aren't these not really types? Especially in a typescript project, the filename seems confusing. Maybe `errortypes.ts` would make more sense, but probably `errorClasses.ts` would be even better.

Reply:
Agree, did not catch that the first time, good eye Andrew!